dogs-initial/src/main/java/com/lambdaschool/dogsinitial/controller/DogController.java

```diff
@@ -34,6 +46,9 @@
             produces = {"application/json"})
     public ResponseEntity<?> getDogDetail(@PathVariable long id) throws ResourceNotFoundException
     {
+        logger.info("/dogs "+id+ " accessed");
+//        MessageDetail message = new MessageDetail("/dogs \" +id+\" accessed ", 1, true);
+//        rt.convertAndSend(DogsinitialApplication.QUEUE_NAME_HIGH, message);
```

Comment:
You can add the date/time of access to your log/message text by concatenating `new Date()` to your log string (ex.: `new Date() + " endpoint accessed."`).

Also, why comment out the lines for sending to your create message queue? It would actually work properly (though instead of sending to the same queue as the "/dogs/dogs" endpoint, you can modify the first parameter of rt.convertAndSend to be a different queue name string instead).